velox/exec/tests/HashTableTest.cpp

```diff
@@ -546,12 +546,13 @@ TEST_P(HashTableTest, mixed6Sparse) {
 TEST_P(HashTableTest, clear) {
   std::vector<std::unique_ptr<VectorHasher>> keyHashers;
   keyHashers.push_back(std::make_unique<VectorHasher>(BIGINT(), 0 /*channel*/));
-
+  core::QueryConfig config({});
```

Comment:
nit: "({})" not needed

Reply:
If I remove it, I will meet:
```
[build] ../velox/expression/tests/ExpressionFuzzer.cpp:183:25: error: no matching function for call to ‘facebook::velox::core::QueryConfig::QueryConfig()’
[build]   183 |       core::QueryConfig config;
```

Reply:
Can you try: `core::QueryConfig config;`?

Reply:
I see. Thanks.

```
  explicit QueryConfig(
      const std::unordered_map<std::string, std::string>& values);
```